### DISCUSSION TITLE
What makes a change be directed to the `20.x` branch?

### DISCUSSION THREAD

**Discussion by elidrissidev** [OpenMage/magento-lts#1839] [Questions & Answers]
Just curious to know, what makes a Pull Request be directed to the `20.x` branch apart from containing non-BC changes?

**Comment by Flyingmana** [comment]
Not directly what is asked, but maybe still usefull.
When there is a PR to the `1.9.4.x` Branch, Iam asking myself this questions:

* is this a bugfix
* is there a big potential for merge conflicts, if its only added to newer branches (codestyle changes)
* is there a notable advantage to have this in all branches (for example github actions, things which make testing and debugging easier)


if non of the points are answered with `yes`, then Iam very likely to move it to the 20.x branch

**Reply by elidrissidev** [reply]
Makes sense.
